packages/helix-shared-all/LICENSE.txt

```diff
@@ -0,0 +1,264 @@
+
```

Comment:
Probably needs to be copied to the root

Reply:
I would keep copies in all of the published packages....